.gitignore

```diff
@@ -1,5 +1,107 @@
-# Ignore temporary packaging folder
-*.egg-info
+# Byte-compiled / optimized / DLL files
+__pycache__/
```

Comment:
Why so many ignore items?

Reply:
I just copied it from another plugin's init, most of the folders and extensions may not appear in this plugin, I can clean it.